packages/terra-button/src/Button.module.scss

```diff
@@ -140,10 +145,14 @@
 
   .icon {
     display: inline-block;
+    line-height: 0;
   }
 
   .icon-svg {
     height: var(--terra-button-icon-height, 1rem);
     width: var(--terra-button-icon-width, 1rem);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      margin-top: var(--terra-button-icon-ms-margin-top, 0.1429em);
+    }
```

Comment:
can remove - adjustments applied elsewhere now so not needed here.

Reply:
Resolved in 5ebdfe6